include/rocksdb/table.h

```diff
@@ -365,6 +365,14 @@ struct BlockBasedTableOptions {
   // This must generally be true for gets to be efficient.
   bool whole_key_filtering = true;
 
+  // If true, detect corruption during (new) Bloom Filter and
```

Comment:
Got you - just saw your #9363

Reply:
> This is an extra check (as in `DisableExtraChecks`) 

The `filter_detect_corruption` is not currently under `DisableExtraChecks` 
https://github.com/facebook/rocksdb/pull/9363/ yet IIUC. Maybe we should remove "(as in DisableExtraChecks)" to avoid illusion that the feature was included. If we ever add this feature under  `DisableExtraChecks` due to performance concern, we list this in the comment of `DisableExtraChecks` as "this includes feature x,y,z and filter_detect_corruption".

Reply:
Good points.

We know this feature has a CPU cost, so I suggest adding it to `DisableExtraChecks` even though this feature is currently disabled by default. Or you can say "(like those in DisableExtraChecks)" or remove the reference to DisableExtraChecks. Any of those is OK.

Reply:
Will remove the reference add it to DisableExtraChecks in another quick PR.